packages/noya-renderer/src/components/SketchFileRenderer.tsx

```diff
@@ -493,6 +516,9 @@ export default memo(function SketchFileRenderer() {
         )}
       </Group>
       <Group transform={screenTransform}>
+        {interactionState.type === 'insertingSymbol' && symbol && (
+          <SketchLayer key={symbol.symbolID} layer={symbol} />
```

Comment:
We may want to render `<SketchArtboardContent />` directly instead of `<SketchLayer />`, so that we don't show the symbol's title

![Screen Shot 2021-07-08 at 12 54 37 PM](https://user-images.githubusercontent.com/1198882/124982535-a98f9500-dfeb-11eb-99ae-d19b96bb846d.png)

Alternately we could instantiate a symbolInstance, but that seems like more work

Reply:
It's easier when using the SketchArtboardContent, it did something weird when using symbolInstance